.venv/project/ecommerce/api/shoppingCart.py

```diff
@@ -0,0 +1,109 @@
+from urllib.error import HTTPError
+from firebase_admin import firestore
+import json
+from requests.exceptions import HTTPError
+from .itembrowsing import get_item_by_ID
+from ..models.items import Item
+
+
+db = firestore.client()
+
+items_ref=db.collection(u'items')
+
+
+def add_item_to_cart(email, itemId, quantity):
```

Comment:
Does itemId return the name and price of the item?

Reply:
the itemId references the item, with an id like '4eyrADdp5Mpy2NhXICGt' and quantity, not its name and price.
the shopping cart only stores items with their itemId, and quantity.
And the function get_items_from_cart returns the items with all their fields like name, price, description...
Does this answer your question?

Reply:
Yeah thanks